### PR TITLE
Update authentication tutorial language

### PR DESCRIPTION
**Pull request by polandtyler**
- Removes an import that may be confusing and is unnecessary for the iOS tutorial
- Corrects the tab reference (docs instead of schema)

### CODE REVIEW

File: docs/source/tutorial/tutorial-authentication.mdx [hunk 1]
```diff
@@ -2,8 +2,6 @@
 title: 7. Enable authentication
 ---
 
-import LoginUISetupPanel from "./components/login_ui_setup_panel.mdx"
-
```
Comment (designatednerd) [inline docs/source/tutorial/tutorial-authentication.mdx:6]: This is an import for our documentation javascript, not for Swift. The instructions for setting up the UI will not render without this (See line 111). Please put this back.

File: docs/source/tutorial/tutorial-authentication.mdx [hunk 2]
```diff
@@ -14,7 +12,7 @@ In this section, you'll add the ability to log in to the example server and obta
 
 A **mutation** is an operation that changes state on your server. In this case, the mutation changes back-end state by creating a session tied to a particular user of your client.
 
-Open [GraphiQL](https://apollo-fullstack-tutorial.herokuapp.com/) and delete anything that's already in the left-hand panel. Open the Schema tab and select the `login` mutation:
+Open [GraphiQL](https://apollo-fullstack-tutorial.herokuapp.com/) and delete anything that's already in the left-hand panel. Open the Docs tab and select the `login` mutation:
```
Comment (designatednerd) [inline docs/source/tutorial/tutorial-authentication.mdx:15]: Good catch! I was using an older version of GraphiQL when I wrote this and that's where they were. I thought I got all these but apparently not - you'll see that in the screenshot it's still showing the Schema tab. Do you mind also updating the screenshot?

**Comment by designatednerd** [comment]
(test failure is a flaky test, I wouldn't worry about that)

File: docs/source/tutorial/tutorial-authentication.mdx (hunk 1 above)
Comment (polandtyler) [reply docs/source/tutorial/tutorial-authentication.mdx:6]: Will do! Thanks for the heads-up. After going back and looking at some of the other commits, I realized my mistake.

File: docs/source/tutorial/tutorial-authentication.mdx (hunk 2 above)
Comment (polandtyler) [reply docs/source/tutorial/tutorial-authentication.mdx:16]: Done & done! Appreciate the feedback!

**Pull request merged**